app/src/main/java/com/ivy/wallet/Constants.kt

```diff
@@ -28,4 +28,7 @@ object Constants {
         "https://github.com/ILIYANGERMANOV/ivy-wallet#contributors-see-graph"
 
     const val USER_INACTIVITY_TIME_LIMIT = 60  //Time in seconds
+
+    const val SWIPE_DOWN_THRESHOLD_OPEN_MORE_MENU = 200
+    const val SWIPE_HORIZONTAL_THRESHOLD = 250
```

Comment:
This constant is local for HomeMoreMenu and isn't global => it shouldn't be here.